ext/AgentsGraphVisualizations/AgentsGraphVisualizations.jl

```diff
@@ -1,20 +1,21 @@
 
 module AgentsGraphVisualizations
 
-using Agents, GraphMakie
+using Agents, Makie, GraphMakie
+const _ABMPlot = Agents.get_ABMPlot_type()
 
-function Agents.agents_graphplot!(
-    abmplot, 
-    graph;
-    node_color, 
-    node_marker, 
-    node_size,
-    graphplotkwargs,
-    edge_color, 
-    edge_width)
-    graphplot!(abmplot, graph; node_color=node_color, node_marker=node_marker, 
-               node_size=node_size, graphplotkwargs..., # must come first to not overwrite lifted kwargs
-               edge_color, edge_width)
+include("src/spaces/graph.jl")

```

Comment:
```suggestion
```

Reply:
Why should this not be necessary? The files have to be included to load the plotting methods for GraphSpace and OpenStreetMapSpace.

When you tested it, did you maybe still have those methods loaded in a running Julia instance? Maybe you forgot to restart Julia to reload the extensions and test whether it works without the include statement?

Reply:
never mind, I now noticed that the package didn't trigger precompilation when I made the change and so I assumed it worked